third_party/move/move-binary-format/src/file_format.rs

```diff
@@ -1456,6 +1482,13 @@ impl SignatureToken {
         }
     }
 
+    /// Returns true if the `SignatureToken` is a function.
```

Comment:
I disagree: off-line discussions will lead to nothing because no one will do the scan across all aptos-core to fix wrong/outdated comments or their style. We should fix these things incrementally as we touch the code.

For this specific issue, I am fine with keeping `` for now. To give you an example:
```
/// `InterpreterImpl` instances can execute Move functions.
///
/// An `Interpreter` instance is a stand alone execution context for a function.
/// It mimics execution on a single thread, with an call stack and an operand stack.
pub(crate) struct InterpreterImpl { ... 
```

Reply:
Anyone who cares can do the scan and fix.  It's much easier to review a style cleanup as an independent PR.